packages/mobile/src/screens/chat-screen/ChatListScreen.tsx

```diff
@@ -110,7 +115,15 @@ export const ChatListScreen = () => {
   const navigation = useNavigation<AppTabScreenParamList>()
   const chats = useSelector(getChats)
   const chatsStatus = useSelector(getChatsStatus)
+  const otherUsers = useSelector(getAllOtherChatUsers)
 
+  // If this is the first fetch, we want to show the loading skeleton that
+  // fades out. Otherwise, we want to show a skeleton in each incoming chat row.
+  const numberOfFetchChatRequests = useSelector(getNumberOfFetchChatRequests)
```

Comment:
I don't think we need this - I think this is inferred by `chats.length > 0`

Reply:
I thought this too but the complicated bit is `chats.length` can be `> 0` but the users haven't loaded yet, and we still want to show the skeleton in that case. But *only* for that case and not subsequent loads.

Reply:
Spoke offline, we're gonna nix this fetch counter and just live with the fading skeleton transitioning into non-fading skeleton with rows popping in. We fetch chats from the root screen so ideally users will never see the fading skeleton anyway. I'll leave the fading skeleton code in bc once we finish the common migration, we can have the `doFetchMoreChats` saga wait for users to load in as well, which will make it more likely to see the fading skeleton.